pkg/util/topology_dumper.go

```diff
@@ -0,0 +1,94 @@
+package util
```

Comment:
This file is used only for testing and we don't want it to be included in the product build. We can have it in some test util files, e.g., utils_test.go.